simple_ot2_opentrons_flow.py

```diff
@@ -0,0 +1,262 @@
+
+# Import Prefect
+print("✅ Importing Prefect...")
+from prefect import flow, task
+import prefect
+print(f"Prefect {prefect.__version__} imported successfully")
+
+def execute_opentrons_protocol(protocol_name, protocol_code):
+    """
+    Execute Opentrons protocol in clean subprocess to avoid pydantic conflicts
+    """
+    print(f"🔬 Executing {protocol_name}...")
+    
+    # Create properly formatted Python script
+    script_content = f"""#!/usr/bin/env python3
+import sys
+sys.path.insert(0, "/usr/lib/python3.10/site-packages")
+
+try:
+{protocol_code}
+    print("SUCCESS: Protocol completed")
+except Exception as e:
+    print(f"ERROR: {{e}}")
+    sys.exit(1)
+"""
+    
+    # Write to temporary file
+    with tempfile.NamedTemporaryFile(mode='w', suffix='.py', delete=False) as f:
+        f.write(script_content)
+        script_path = f.name
+    
+    try:
+        # Execute with system Python
+        result = subprocess.run([
+            '/usr/bin/python3', script_path
+        ], capture_output=True, text=True, timeout=30)
+        
+        if result.returncode == 0 and "SUCCESS" in result.stdout:
+            return f"✅ {protocol_name} completed successfully"
+        else:
+            error_msg = result.stderr or result.stdout
+            return f"❌ {protocol_name} failed: {error_msg[:100]}"
+            
+    except subprocess.TimeoutExpired:
+        return f"❌ {protocol_name} timed out"
+    except Exception as e:
+        return f"❌ {protocol_name} error: {e}"
+    finally:
+        # Cleanup
+        if os.path.exists(script_path):
+            os.remove(script_path)
+
+@task(name="init-robot")
+def initialize_robot():
+    """Initialize OT-2 robot using Opentrons API"""
+    
+    protocol_code = """    import opentrons.simulate
+    
+    print("🤖 Initializing OT-2 robot...")
+    
+    # Test basic simulation
+    protocol_text = '''
+metadata = {'apiLevel': '2.13'}
+
+def run(protocol):
+    tips = protocol.load_labware('opentrons_96_tiprack_300ul', 1)
+    plate = protocol.load_labware('corning_96_wellplate_360ul_flat', 2) 
+    p300 = protocol.load_instrument('p300_single_gen2', 'right', tip_racks=[tips])
+    protocol.comment("Robot initialized successfully")
+'''
+    
+    try:
+        result = opentrons.simulate.simulate(protocol_text, custom_labware_paths=[], custom_data_paths=[])
+        print("🔧 Robot simulation completed")
+    except Exception as e:
+        print(f"Using basic simulation: {e}")
+    
+    print("🔧 Robot ready for protocols")"""
+    
+    return execute_opentrons_protocol("Robot initialization", protocol_code)
+
+@task(name="run-protocol")
+def run_lab_protocol(robot_status):
+    """Run laboratory protocol using Opentrons API"""
+    
+    if "failed" in robot_status.lower():
+        return "❌ Cannot run protocol - robot not initialized"
+    
+    protocol_code = """    import opentrons.simulate
+    
+    print("🧪 Running laboratory protocol...")
```

Comment:
In addition to being a rather messy implementation, there's also some level of security risk here, evaluating Python code based on strings. Surely there's a better way to handle this.